tests/Client/UpdaterTest.php

```diff
@@ -100,6 +100,29 @@ protected function getSystemInTest() : Updater
         return $updater;
     }
 
+    /**
+     * Tests that TUF will transparently verify downloaded target hashes.
+     *
+     * @covers ::download
+     *
+     * @return void
+     */
+    public function testVerifiedDownload(): void
+    {
+        $fixturesSet = 'TUFTestFixtureSimple';
+        $this->localRepo = $this->memoryStorageFromFixture($fixturesSet, 'tufclient/tufrepo/metadata/current');
+        $this->testRepo = new TestRepo($fixturesSet);
+        $updater = $this->getSystemInTest();
+
+        $this->testRepo->repoFilesContents['testtarget.txt'] = 'testtarget.txt';
+        $updater->download('testtarget.txt')->wait();
```

Comment:
a couple things here.
1. It confusing looking at the test that file contents are the same as the file. 
I believe this comes from `generate_fixtures.py`
`targetfile.write(filename)`
But I think we should up this in this issue or I think it will be forgotten about. Something like
`targetfile.write("Contents: " + filename)`

2. Does this actually test that we actually downloaded anything? I mean `wait()` could return anything or nothing and the test would still pass, correct?

Reply:
1. I decided instead to directly read the contents of `testtarget.txt`. Hopefully this will make the intention clearer.
2. Added an assertion to prove that we actually get the contents of the file.